service-base/src/main/java/org/eclipse/hono/service/management/credentials/CredentialsManagementHttpEndpoint.java

```diff
@@ -71,6 +73,10 @@ public void addRoutes(final Router router) {
         final String pathWithTenantAndDeviceId = String.format("/%s/:%s/:%s",
                 getName(), PARAM_TENANT_ID, PARAM_DEVICE_ID);
 
+
+        // Add CORS handler
+        router.route(pathWithTenantAndDeviceId).handler(createCorsHandler(config.getCorsAllowedOrigin(), EnumSet.of(HttpMethod.PUT)));
```

Comment:
Shouldn't this be PUT and GET?

Reply:
I think so too. The GET requests supported by the Management API are never *simple* because they always include the *ETag* header and will usually have a content type that is not among the *simple* ones ...